governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
```

Comment:
> There could be a high level summary here, e.g.

## Summary

The community works in special interest groups (SIGs), who form around a common topic.

SIGs are operated by consensus and can self-organize.
A common organizational structure might be to appoint a chair and vice-chair by consensus, who organize and run meetings, and take charge of getting the group to consensus.

The overall project is operated by all SIGs in consensus.
The future roadmap includes the idea of committees, which are elected or selected bodies that operate outside of the SIG structure, and may direct the project above and beyond the SIGs.
For example, a future Technical Steering Committee might decide all overall technical directions, based on the advice and consent of the SIGs.

To start, we are initiating the Community SIG and the Operations SIG, all parts of the project fall under these SIGs.
The SIGs then self-organize and decide to sub-divide into further SIGs, and to form subprojects within one or more SIGs to actually do things.

The subprojects are where the work happens, the SIGs organize and guide their subprojects.

Reply:
...common organizational structure ~~might be~~ **is** to appoint a chair and vice-...

And ACK on the *Summary*. Just add it as an additional commit to the PR and we can comment on the section content

Reply:
The community works in special interest groups (SIGs), that form around a common topic.

SIGs are operated by consensus and can self-organize.
A common organizational structure might be to appoint a chair and vice-chair by consensus who organize and run meetings and take charge of getting the group to consensus.

The overall project is operated by all SIGs in consensus.
The future roadmap includes the idea of committees, which are elected or selected bodies that operate outside of the SIG structure and may direct the project above and beyond the SIGs.
For example, a future Technical Steering Committee might decide all overall technical directions based on the advice and consent of the SIGs.

To start, we are initiating the Community SIG and the Operations SIG. All parts of the project fall under these SIGs.
The SIGs then self-organize and decide to sub-divide into further SIGs and to form subprojects within one or more SIGs to actually do things.

The subprojects are where the work happens, and the SIGs organize and guide their subprojects.

Reply:
The summary should perhaps start with (a tl;dr of) the why and not with the how

Reply:
> ...common organizational structure ~might be~ **is** to appoint a chair and vice-...

I get your intent here, I made that change but also expanded the paragraph just a bit to clarify the "one common structure" is not a dictate.

The key is to let SIGs self-organize rather than restrict them to start, and only restrict structures that prove problematic (rather than are predicted to be problematic.)

> And ACK on the _Summary_. Just add it as an additional commit to the PR and we can comment on the section content

And also AIUI, we can provide fixes as comments, which can then be merged like they were a direct commit.

Reply:
Updated in d1adfd2